From 43dc753446d2e8e8b611b3b75452166a4a026c5b Mon Sep 17 00:00:00 2001
From: John Veitch <john.veitch@ligo.org>
Date: Mon, 13 Dec 2021 17:23:33 +0000
Subject: [PATCH 1/2] lalinference.wrapper: Update import of MutableMapping

---
 lalinference/python/lalinference/wrapper.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lalinference/python/lalinference/wrapper.py b/lalinference/python/lalinference/wrapper.py
index 6c7a8f5126..b9b7fdf2a1 100644
--- a/lalinference/python/lalinference/wrapper.py
+++ b/lalinference/python/lalinference/wrapper.py
@@ -2,7 +2,7 @@ import lal
 import lalinference as li
 import collections
 
-class LIVariablesWrap(collections.MutableMapping):
+class LIVariablesWrap(collections.abc.MutableMapping):
     def __init__(self,init=None):
         """
         Wrapper to present a LALInferenceVariable as a dict.
-- 
GitLab


From 1f88608f8d36e9784da9ffcae92696d05ffd04f2 Mon Sep 17 00:00:00 2001
From: John Veitch <john.veitch@ligo.org>
Date: Tue, 14 Dec 2021 10:27:53 +0000
Subject: [PATCH 2/2] import collections.abc

---
 lalinference/python/lalinference/wrapper.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lalinference/python/lalinference/wrapper.py b/lalinference/python/lalinference/wrapper.py
index b9b7fdf2a1..79c04d2e77 100644
--- a/lalinference/python/lalinference/wrapper.py
+++ b/lalinference/python/lalinference/wrapper.py
@@ -1,6 +1,6 @@
 import lal
 import lalinference as li
-import collections
+import collections.abc
 
 class LIVariablesWrap(collections.abc.MutableMapping):
     def __init__(self,init=None):
-- 
GitLab

